inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -487,17 +487,20 @@ CNNLayerPtr InferenceEngine::details::CNNLayerCreator::create() {
     return res;
 }
 
-std::shared_ptr<CNNNetworkImpl> convertFunctionToICNNNetwork(const std::shared_ptr<const ::ngraph::Function>& graph, const ICNNNetwork &network) {
+std::shared_ptr<CNNNetworkImpl> convertFunctionToICNNNetwork(const std::shared_ptr<const ::ngraph::Function> &graph,
+                                                             const ICNNNetwork &network,
+                                                             bool keep_constant_inputs) {
     IE_PROFILING_AUTO_SCOPE(convertFunctionToICNNNetwork)
     const auto createCNNLayer = [](const std::shared_ptr<::ngraph::Node> &node) -> CNNLayerPtr {
-        class NGraphCNNLayer: public CNNLayer {
+        class NGraphCNNLayer : public CNNLayer {
```

Comment:
Could you please revert code style related changes?

Reply:
What code style do you mean?

Reply:
All this spaces changes like in this line. They disturb reviewing actual changes.

Reply:
I have found some spaces and delete them. May be these "extra" spaces were created by auto-format tool in CLion.

Reply:
I would recommend to disable this auto-formatting in IDE for existing code, if possible. It might introduce a lot of changes not related to the PR scope, making it harder to review.

Reply:
agree to revert code style

Reply:
Done.